ddtrace/internal/datastreams/__init__.py

```diff
@@ -1,14 +1,23 @@
-from . import kafka  # noqa:F401
+from ddtrace import config
+from ddtrace import tracer
 
+from ...internal.utils.importlib import require_modules
 
+
+required_modules = ["confluent_kafka"]
 _processor = None
 
+if config.data_streams_enabled:
+    with require_modules(required_modules) as missing_modules:
+        if not missing_modules:
+            from . import kafka  # noqa:F401
+
 
 def data_streams_processor():
     from . import processor
 
     global _processor
-    if not _processor:
-        _processor = processor.DataStreamsProcessor()
+    if config.data_streams_enabled and not _processor:
+        _processor = processor.DataStreamsProcessor(tracer._agent_url)
```

Comment:
is this a breaking change?  We're moving from "attach your own processor" to "use the global processor".   I *feel* like its not a breaking change, because the stats should end up the same, but I'd like another opinion.

Reply:
It doesn't look to me like a breaking change.